pkg/storaged/nfs-mounts.py

```diff
@@ -215,6 +215,9 @@ def fuser(entry):
     mount_point = entry["fields"][1]
     results = { }
 
+    def get_cmdline(pid):
+        return " ".join(open("/proc/%s/cmdline" % pid).read().split("\0"))
```

Comment:
Some command lines can get really long and thus break the dialog layout. For example:

    /usr/bin/dbus-daemon --session --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only

but this isn't even half as bad as current firefox, which encodes its *entire* configuration on the CLI and has binary junk in it.

    ❱❱❱ pgrep -af firefox | wc -c
    7712

This really shouldn't be dumped verbatim here. Perhaps show the command line if it's < 20 characters, and fall back to just argv[0] (i. e. the first element in this array) otherwise?

Reply:
Yeah, I decided to ignore this... will fix.

Reply:
Looks like this now:
![screenshot from 2018-03-26 12-28-02](https://user-images.githubusercontent.com/3228183/37898315-29c2058a-30f1-11e8-862c-6159755e5709.png)